lib/go/edgecontext/edgecontext_test.go

```diff
@@ -33,12 +33,14 @@ const (
 )
 
 const (
-	expectedCountryCode = "OK"
-	expectedDeviceID    = "becc50f6-ff3d-407a-aa49-fa49531363be"
-	expectedLoID        = "t2_deadbeef"
-	expectedOrigin      = "baseplate"
-	expectedSessionID   = "beefdead"
-	expectedRequestID   = "2adaff94-9067-4de0-a00b-79fded5cff9e"
+	expectedCountryCode  = "OK"
+	expectedLanguageCode = "EN"
```

Comment:
will this actually be capitalized? I think conventionally we don't capitalize language code.